packages/rum/src/viewCollection.ts

```diff
@@ -77,8 +93,9 @@ function newView(
     userActionCount: 0,
   }
   let documentVersion = 0
+  let loadDuration: number
 
-  viewContext = { id, location, sessionId: session.getId() }
+  viewContext = { id, location, loadType, sessionId: session.getId() }
```

Comment:
why do we need that loadType in the view context?

Reply:
this exported view context is used for the test and allows to expect the proper loadType for each scenarios

Reply:
What about using lifecycle events like the other tests to avoid leaking unnecessary data in this context?

Reply:
We already have a VIEW_COLLECTED event that notify the View updates but using this event would require to mock the view notification update process. The existing path change tests would not be straightforward anymore. Also creating a new event for these tests only would not be ideal as we would have duplicates.
For the understanding of the tests, keeping viewContext seems better.

Reply:
I would be in favor of adapting the tests to the implementation rather than the other way around.
I am not in favor of adding direct dependency between viewCollection and userActionCollection when lifecycle purpose it to avoid that.